examples/autogluon_tabular_tutorial.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
```

Comment:
Could you fix this ModuleNotFoundError?

In general, this entire notebook should be run with nice output before being comitted right?

Reply:
I removed the notebook and instead expanded the tests: those example notebooks are at risk of becoming outdated, especially when they depend on newer modules

Reply:
But will there be some sort of tutorial somewhere on how to use this model? 
Otherwise I worry all the various options made available here will be quickly be forgotten.

Reply:
Yes, we can add a notebook to the website: those are run as part of the build, so we make sure not to break them or to update them